package.json

```diff
@@ -22,8 +22,10 @@
   },
   "dependencies": {
     "cordova-common": "^2.2.0",
+    "fs-extra": "^6.0.1",
     "q": "^1.4.1",
-    "shelljs": "^0.7.0"
+    "shelljs": "^0.7.0",
```

Comment:
Are we able to remove shelljs entirely from the package.json?

Reply:
Of course. Good catch. Will update this to remove it.

Embarrassing oversight given the title of the PR :sweat_smile: